torch/csrc/distributed/c10d/UCCUtils.hpp

```diff
@@ -188,6 +188,27 @@ ucc_status_t oob_allgather_test(void* req);
 
 ucc_status_t oob_allgather_free(void* req);
 
+// trim: remove spaces before and after the string view
+// implementation borrowed from https://stackoverflow.com/a/17976541
+inline c10::string_view trim(c10::string_view s) {
```

Comment:
just curious, does PyTorch have a way to write some unit tests for util methods like these?

Reply:
I am also curious the reason to have a trim function. I believe we usually do not allow spaces in the env. variables.

Reply:
It is mostly to allow the user to specify something like
```
TORCH_UCC_BLOCKING_WAIT ="allreduce, send, recv"
```
The comma is still needed.

Reply:
> just curious, does PyTorch have a way to write some unit tests for util methods like these?

Yes, there are cpp tests.

Reply:
Added cpp tests